android/gradle.properties

```diff
@@ -1,4 +1,4 @@
-propMinSdkVersion=21
+propMinSdkVersion=23
```

Comment:
You want to cancel Android 5.0 and 5.1 support and make Android 6.0 minimal requirement. What block you from using `propMinSdkVersion=21`?